core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -757,6 +769,18 @@ protected function createFooterScripts($objLayout)
 			}
 		}
 
+		$arrJsAssets = \StringUtil::deserialize($objLayout->jsAssets);
+
+		// External style sheets
```

Comment:
"External **JavaScripts**", isn't it?

Reply:
Fixed in 1ba5f25f72a0343804fdf5dab1a823227dcb1298